.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file for?

Reply:
It was auto-generated by IntelliJ so I just decided to leave it in. After a search though it seems I can probably remove it without issue